SQL/paradise_schema.sql

```diff
@@ -446,10 +446,13 @@ DROP TABLE IF EXISTS `erro_watch`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `erro_watch` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
   `ckey` varchar(32) NOT NULL,
   `reason` text NOT NULL,
-  PRIMARY KEY (`id`)
+  `timestamp` datetime NOT NULL,
+  `adminckey` varchar(32) NOT NULL,
```

Comment:
Primary key is best used for whatever column can be said to uniquely identify a row in a table. In our system, because each ckey can only have one active memo, we decided that ckey was that row.

People generally use auto increment ID fields because they see most tables use them, but an ID field isn't always the best option, (it's never a wrong one, but it's not always the best one either.)

Primary key also has the best performance in WHERE clauses and literally everything memo wise selects by ckey.

All the more reason.

Removing the ID column was my idea, because it removes some unneeded bloat in the table, and was only added because hippie station just made the table match the others.